RRTPlannerROS: {
  goal_tolerance: 0.20,
  angle_tolerance: 0.1,
  rrt: {
    step: 0.10,
    min_num_nodes: 200,
    max_num_nodes: 5000,
  }
}
